pkg/util/utils.go

```diff
@@ -32,6 +32,16 @@ const (
 	downloadURL = "https://storage.googleapis.com/minikube/releases/%s/minikube-%s-amd64%s"
 )
 
+// ValidateString returns index, true if a string exists in []string
+func ValidateString(input string, validOptions []string) (int, bool) {
```

Comment:
@medyagh I've created this util so that strings can be validated in the entire project if need arises.

Reply:
plz avoid using util package, we are trying to get rid of util package, define this func in the onlly place that was used. (close to where it was used)

Reply:
plz use a simple look up,
first make the input lowercase and then check if it is in this slice cruntime.ValidOptions()
and if it is in the slice, then return True, otherwise return False